CHANGELOG.md

```diff
@@ -1,5 +1,110 @@
 # Changelog
 
+## v6.0.0-beta.8 - 2017-10-05
+
+### Breaking change
+
+* Changed `new-service` to return error when given unsupported startuptype and fixed set-service failing test (#4802)
+* Allow * to be used in registry path for `remove-item` (#4866)
+* Remove unsupported -showwindow switch (#4903)
+* Fix incorrect position of a parameter which resulted in the args passed as input instead of as args for `InvokeScript()` (#4963)
+
+### Engine updates and fixes
+
+* Make calls to void CodeMethod work (#4850) (Thanks @powercode)
+* Get 'PSVersion' and 'GitCommitId' from the 'ProductVersion' attribute of assembly (#4863) (Thanks @iSazonov)
+* Fix `powershell -version` and help to align with other tools (#4958 & #4931) (Thanks @iSazonov)
+* Load the assembly with 'Assembly.LoadFrom' before 'Assembly.Load' when the file path is given (#4196)
+* Add a generic file watcher function in HelpersCommon.psm1 (#4775)
+* Update old links and fix broken links in host-powershell/README.md (#4877)
+* Fix importing remote module using version filters and added tests (#4900)
+* Enable transcription of native commands on non-Windows (#4871)
+* Add a new line to 'CommandNotFoundException' error string (#4934 & #4991)
+* Fix for Linux platform PowerShell exit on error during SSH remoting connection (#4993)
+
+### General cmdlet updates and fixes
+
+* Added Remove-service to Management module (#4858) (Thanks @joandrsn)
+* Added functionality to set credientials on `Set-Service` command (#4844) (Thanks @joandrsn)
+* `Select-String` changes:
+    * Exclude directories discovered from `-Path` (#4829) (Thanks @iSazonov)
+    * Support more argument completion scenarios by adding `ArgumentCompletionsAttribute` (#4835) (Thanks @iSazonov)
+* Exclude `-Comobject` parameter of `New-Object` on unsupported platforms (#4922) (Thanks @iSazonov)
+* Updated default `ModuleVersion` in `New-ModuleManifest` to `0.0.1` to align with SemVer (#4842) (Thanks @LDSpits)
+* Add Multipart Support to Web Cmdlets (#4782) (Thanks @markekraus)
+* Initialize Web Cmdlets Headers Dictionary Only Once (#4853) (Thanks @markekraus)
```

Comment:
Maybe we shouldn't use a capital letter for the first letter for "headers dictionary only once".